File: Data Structures & Algorithms/climbing-stairs/submission-0.cpp
class Solution {
public:
    int climbStairs(int n) {
       if(n<3)
       return n;
       int first=1;
       int second=2;
       int res=0;
       for(int i=2;i<n;i++)
       {
        res=first+second;
        first=second;
        second=res;
       } 
       return res;
    }
};

File: Data Structures & Algorithms/pow-x-n/submission-0.py
class Solution:
    def myPow(self, x: float, n: int) -> float:
        res=1
        if x == 0:
            return 0
        if n == 0:
            return 1

        for i in range(abs(n)):
            res *= x
        return res if n >= 0 else 1 / res
        